fundamental-concepts/template-authorization/2_parties.md

```diff
@@ -0,0 +1,33 @@
+Like the UNIX file system has `users`, every DAML ledger has a concept of identity.
+
+Because the representation of identities are different from one ledger to another, DAML has an
+identity abstraction. This is the `Party` type. So for example if we look at the `User` template of
+the `create-daml-app` package in `daml/User.daml`{{open}}, the line
+
+```
+username: Party
+```
+
+says that the field `username` of the `User` contract will be stored on the used backing ledger in
+whatever representation that ledger uses for identities. This could be public keys or simply a
```

Comment:
```suggestion
says that the field `username` of the `User` contract will be stored on the underlying ledger in
whatever representation that ledger uses for identities. This could be public keys or simply a
```
Just a suggestion. Another option would be to just write "used ledger in whatever...". Was confused with the word `backing`